drivers/console/ipm_console_receiver.c

```diff
@@ -10,11 +10,12 @@
 
 #include <kernel.h>
 #include <ring_buffer.h>
-#include <misc/printk.h>
 #include <stdio.h>
 #include <ipm.h>
 #include <console/ipm_console.h>
 #include <misc/__assert.h>
+#include <misc/printk.h>
```

Comment:
Delete?

Reply:
actually printk still being used here in a different context, still keeping it for now to avoid breakage.